src/views/connecting/Connecting.js

```diff
@@ -123,14 +137,24 @@ export const Connecting = (props) => {
       // send member connected post message before analytic event, this allows clients to show their own "connected" window before the connect complete step.
       if (uiMessageVersion === 4) {
         const event = {
-          user_guid: currentMember.user_guid,
           member_guid: currentMember.guid,
+          user_guid: currentMember.user_guid,
+          raw_status: currentMember.raw_status,
+          selected_account_id: currentMember.selected_account_id,
         }
 
         if (currentMember.aggregator) {
           event.aggregator = currentMember.aggregator
         }
 
+        if (currentMember.selected_account_id) {
+          event.selected_account_id = currentMember.selected_account_id
+        }
+
+        if (currentMember.raw_status) {
+          event.raw_status = currentMember.raw_status
+        }
+
```

Comment:
It looks like the new key/value pairs are added to the object, then added again in the if statements.